docs/en/sql-reference/sql-statements/data-definition/ALTER_TABLE.md

```diff
@@ -10,11 +10,13 @@ Modifies an existing table, including:
 
 - [Rename table, partition, index](#rename)
 - [Modify table comment](#alter-table-comment-from-v31)
-- [Atomic swap](#swap)
-- [Add/delete partitions and modify partition attributes](#modify-partition)
-- [Schema change](#schema-change)
+- [Modify partitions (add/delete partitions and modify partition attributes)](#modify-partition)
+- [Modify bucketing method and the number of buckets](#modify-bucketing-method-and-the-number-of-buckets-from-v32)
```

Comment:
![image](https://github.com/StarRocks/starrocks/assets/57167462/a7dabc3d-ef07-475e-ab71-b7f5289aeb58)
Add "the" preceding "bucketing method".